spec/features/user_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :feature do
+  describe 'Index page' do
+    before(:all) do
+      visit users_path
+      @users = User.all
+    end
+
+    it 'should see the username of other users' do
+      @users.each do |user|
+        expect(page).to have_content(user.Name)
+      end
+    end
+
+    it 'should see the profile picture of each user' do
+      @users.each do |user|
+        expect(page).to have_css("img[src*='#{user.Photo}']")
+      end
+    end
+
+    it 'should see the numer of posts for each user' do
+      @users.each do |user|
+        expect(page).to have_content(user.PostsCounter)
+      end
+    end
+
+    it 'should redirect to user\'s show page' do
+      @users.each do |user|
+        find_link(user.Name).click
+        expect(page).to have_current_path(user_path(user))
+      end
+    end
+  end
```

Comment:
- [ ] As suggested by the previous reviewer, 
- please kindly include a test to assert that a user can see the number of likes a post has.
- please also include a test that asserts that you can see the username of each commenter

Reply:
@AnselemOdims if you check the project requirement, you'll see i have implemented all the necessary testing i need to do, you just came up with what is not in the requirement, and with what the previous reviewer asked me to do i have done that and yours does not go in line with what the previous reviewer asked me to do, 
Kindly check project requirement for peruse before you request for changes 
Thanks.